vagrant_setup/gradle.sh

```
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+wget https://download.java.net/java/GA/jdk17.0.2/dfd4a8d0985749f896bed50d7138ee7f/8/GPL/openjdk-17.0.2_linux-x64_bin.tar.gz
+tar xvf openjdk-17.0.2_linux-x64_bin.tar.gz
+sudo mv jdk-17.0.2/ /opt/jdk-17/
+
+sudo echo 'export JAVA_HOME=/opt/jdk-17' >> /etc/profile
```

Shellcheck (0.10.0)

[warning] sudo doesn't affect redirects. Use .. | sudo tee -a file

(SC2024)